app/src/main/java/com/brandon/forzenbook/repository/ForzenRepositoryImpl.kt

```diff
@@ -87,5 +106,8 @@ class ForzenRepositoryImpl(
         const val DATABASE_ERROR_ACCESS = "Error Accessing Database."
         const val FAILED_ENTITY_CONVERT = "Could Not Convert Response to Entity."
         const val RESPONSE_CODE_ERROR = "Unexpected Response Code."
+
+        const val USER_DUPLICATE_CODE = 409
+        const val USER_DUPLICATE_LOG_MESSAGE = "User Already Exists."
```

Comment:
If you plan to change how this is shown to the user in a later PR, put a TODO with ticket

Reply:
added a TODO